Source/Core/InputCommon/ControllerInterface/DualShockUDPClient/DualShockUDPClient.cpp

```diff
@@ -26,6 +26,8 @@
 #include "InputCommon/ControllerInterface/ControllerInterface.h"
 #include "InputCommon/ControllerInterface/DualShockUDPClient/DualShockUDPProto.h"
 
+#define DUALSHOCKUDP_SOURCE_NAME "DSUClient"
```

Comment:
```suggestion
constexpr std::string_view DUALSHOCKUDP_SOURCE_NAME = "DSUClient"
```

This should also be moved down into the namespace.

Reply:
True, though I copied the way from DInput.h, which declared its name outside of the namespace and as a define.

Reply:
fixed